packages/iov-ethereum/src/ethereumconnection.ts

```diff
@@ -133,8 +155,10 @@ export class EthereumConnection implements AtomicSwapConnection {
       this.socket.connect();
     }
 
-    const erc20Tokens = options.erc20Tokens ? options.erc20Tokens : new Map();
+    const atomicSwapEtherContractAddress = options.atomicSwapEtherContractAddress || ("0x0" as Address);
```

Comment:
Why this fallback when `this.atomicSwapEtherContractAddress` is optional? I prefer `undefined` over the dummy value.